core/src/main/java/bio/terra/pearl/core/service/export/ExportOptions.java

```diff
@@ -1,13 +1,16 @@
 package bio.terra.pearl.core.service.export;
 
-import bio.terra.pearl.core.service.export.ExportFileFormat;
 import lombok.Builder;
 
-public record ExportOptions (boolean splitOptionsIntoColumns, boolean stableIdsForOptions, boolean onlyIncludeMostRecent,
-                             ExportFileFormat fileFormat,
-                             Integer limit) {
+public record ExportOptions(
```

Comment:
this constructor is getting unwieldy with booleans.  How about we convert this to a class so we can have defaults and use a lombok builder?